docs/quantum_keycodes.md

```diff
@@ -16,6 +16,11 @@ On this page we have documented keycodes between `0x00FF` and `0xFFFF` which are
 |`KC_GESC`      |`GRAVE_ESC`|Escape when tapped, <code>&#96;</code> when pressed with Shift or GUI|
 |`KC_LSPO`      |           |Left Shift when held, `(` when tapped                                |
 |`KC_RSPC`      |           |Right Shift when held, `)` when tapped                               |
+|`KC_LCPO`      |           |Left Control when held, `(` when tapped                              |
```

Comment:
These are technically not correct, as `LCPO_KEYS` etc. define the same hold and tap modifier. They should all be Shift if this is the case.

https://github.com/qmk/qmk_firmware/blob/62ba66d61821fec6a5ad3bdccdf738e15e082461/quantum/process_keycode/process_space_cadet.c#L61-L75

Reply:
...good point

Reply:
Fixed